### DISCUSSION TITLE
Options Trading Strategies research group

### DISCUSSION THREAD

**Discussion by gugavaro** [DaveSkender/Stock.Indicators#1033] [Show and tell]
Hey all, I have been using Stock.Indicators library on my toolbox to analyze and generate trade alerts/ideas and it is a wonderful tool.
I was wondering how people here are using it and if you all also automate your trades?
Would be nice to exchange ideas of how you all are using the indicators to help you in your strategies.
I have a private slack channel where myself and a couple of friends discuss strategies and backtest, more focus on options trading but not limited to, let me know if you have been trying out different strategies and would be interested in exchange ideas on it.

thanks,
Gustavo.

**Comment by phivanloi** [comment]
Hi gugavaro
I have a center that analyzes and processes buy/sell signals for Vietnam's securities. I am still developing and perfecting my investment strategy and backtesting it. Happy to share, learn more of your strategies.
